sdk/identity/identity/test/manual/src/index.tsx

```diff
@@ -164,7 +164,7 @@ function ClientDetailsEditor({ clientDetails, onSetClientDetails }: ClientDetail
         <br />
         <h4>Authentication flow</h4>
         <Radio
-          values={["implicit-grant", "auth-code"]}
+          values={["auth-code"]}
```

Comment:
Since there's just auth code flow maybe we don't need this radio button at all anymore?